projects/views.py

```diff
@@ -1,3 +1,11 @@
-# from django.shortcuts import render
+from .models import Project
 
-# Create your views here.
+from django.views.generic.list import ListView
+
+
+class ProjectListView(ListView):
+    model = Project
+
+    def get_context_data(self, **kwargs):
+        context = super(ProjectListView, self).get_context_data(**kwargs)
```

Comment:
if you are not doing anything custom here eg. modifying the context, you can skip the method altogether, inheritance will do what you are doing.